packages/mdc-chips/chip-set/test/component.test.ts

```diff
@@ -46,6 +46,59 @@ const getFixture = () => {
   return el;
 };
 
+const getFixtureWithTrailingAction = () => {
```

Comment:
Can we make a foundation test of `handleChipRemoval` instead?

Something like:

- mock adapter chipCount to return 0
- call handleChipRemoval of last remaining chip
- check that `adapter.focusChipTrailingActionAtIndex` isn't called